### ISSUE TITLE
[BottomSheetDialogFragment] AppTheme still does not get applied

### ISSUE THREAD

**Issue by franvis** [material-components/material-components-android#894]
**Description:** I saw this other issue closed/fixed (https://github.com/material-components/material-components-android/issues/99) but I still cannot achieve to have the AppTheme applied in my BottomSheetDialogFragment.

**Expected behavior:** 
AppTheme being applied correctly to my BottomSheetDialogFragment. I still have to do the workaround of:
```val contextThemeWrapper = ContextThemeWrapper(activity, R.style.AppTheme)
        return inflater.cloneInContext(contextThemeWrapper)
            .inflate(R.layout.view_my_bottom_sheet_dialog_fragment, container, false)
```
to make it work.

**Source code:** The code snippet which is causing this issue. Please consider attaching a minimal sample app that reproduces the issue.

**Android API version:** 29

**Material Library version:** 
1.2.0-alpha03

**Comment by aouerfelli** [comment]
The commit (eefb5d4d778e15f5a0bce6e09f5257c15ede4163) that fixes this issue was pushed to master after 1.2.0-alpha04, so it'll be included in the next release.

**Comment by ikim24** [comment]
Please reopen if the next alpha doesn't fix this for you.

**Comment by HCTwist** [comment]
This still doesn't seem to be fixed post 1.2.0-alpha04, anyone else had troubles with this?

**Comment by matiasmb** [comment]
this still happen with the 1.4.0-rc01, can we reopen this?
